src/components/Home/Home.tsx

```diff
@@ -1,20 +1,27 @@
 import React, { useState, useEffect } from 'react';
-
+import bkg_img from '../../images/home/Big Shoes - Standing Pose.png'
```

Comment:
This file contains HTML for the Landing Page

Reply:
This is the whole component. React essentially creates components that are functions containing an element's functionality, and returns an HTML element.  the element in the return block is actually in JSX which is JS but emulates HTML and complies as html.